core/src/main/java/cz/o2/proxima/repository/EntityDescriptor.java

```diff
@@ -114,6 +114,21 @@ static Builder newBuilder() {
                 String.format("Unable to find attribute [%s] of entity [%s].", name, getName())));
   }
 
+  /** @return {@code true} if the entity contains any attribute that has transactions enabled. */
+  default boolean isTransactional() {
+    return getAllAttributes()
+        .stream()
+        .anyMatch(attr -> attr.getTransactionMode() != TransactionMode.NONE);
+  }
+
+  /**
+   * @return {@code true} if this is system entity. System entities are created by the platform
+   *     itself and should be (directly) accessed by users.
+   */
+  default boolean isSystemEntity() {
```

Comment:
Q: is other entities validated by this? Maybe we should change also `ENTITY_NAME_PATTERN`

Reply:
Hm, I wanted to do this, but it seems like a forget. Will do this in `validateEntityName`